5/README.md

```diff
@@ -0,0 +1,393 @@
+    - [StaticMenu.ApothecaryEntry](#opencannabis.products.menu.StaticMenu.ApothecaryEntry)
+    - [StaticMenu.CartridgesEntry](#opencannabis.products.menu.StaticMenu.CartridgesEntry)
+    - [StaticMenu.EdiblesEntry](#opencannabis.products.menu.StaticMenu.EdiblesEntry)
+    - [StaticMenu.ExtractsEntry](#opencannabis.products.menu.StaticMenu.ExtractsEntry)
+    - [StaticMenu.FlowersEntry](#opencannabis.products.menu.StaticMenu.FlowersEntry)
+    - [StaticMenu.MerchandiseEntry](#opencannabis.products.menu.StaticMenu.MerchandiseEntry)
+    - [StaticMenu.PlantsEntry](#opencannabis.products.menu.StaticMenu.PlantsEntry)
+    - [StaticMenu.PrerollsEntry](#opencannabis.products.menu.StaticMenu.PrerollsEntry)
+
+
+----
+
+## Protocol Definition
+### `opencannabis.menu`
+Specifies a menu, menu data structure, contents, products and so on
+
+{% nomnoml %}
+
+#fill: #d5e7ee; #8ebff2
+
+[Menu
+  | metadata:Metadata
+  | payload:SectionedMenu
+  | menu:StaticMenu]
+
+[MenuProduct
+  | key:opencannabis.base.ProductKey
+  | apothecary:opencannabis.products.Apothecary
+  | cartridge:opencannabis.products.Cartridge
+  | edible:opencannabis.products.Edible
+  | extract:opencannabis.products.Extract
+  | flower:opencannabis.products.Flower
+  | merchandise:opencannabis.products.Merchandise
+  | plant:opencannabis.products.Plant
+  | preroll:opencannabis.products.Preroll]
+
+[MenuSettings
+  | full:bool
+  | keys_only:bool
+  | snapshot:opencannabis.crypto.primitives.integrity.Hash
+  | fingerprint:opencannabis.crypto.primitives.integrity.Hash
+  | section:section.Section]
+
+[Metadata
+  | scope:string
+  | version:uint64
+  | status:Status
+  | flags:Flag
+  | published:opencannabis.temporal.Instant
+  | settings:MenuSettings]
+
+{% endnomnoml %}
+
+{% nomnoml %}
+
+#fill: #d5e7ee; #8ebff2
+[SectionData
+  | count:int32
+  | section:section.SectionSpec
+  | product:MenuProduct]
+
+[SectionedMenu
+  | count:int32
+  | payload:SectionData]
+
+{% endnomnoml %}
+
+{% nomnoml %}
+
+#fill: #d5e7ee; #8ebff2
+[StaticMenu
+  | apothecary:StaticMenu.ApothecaryEntry
+  | cartridges:StaticMenu.CartridgesEntry
+  | edibles:StaticMenu.EdiblesEntry
+  | extracts:StaticMenu.ExtractsEntry
+  | flowers:StaticMenu.FlowersEntry
+  | merchandise:StaticMenu.MerchandiseEntry
+  | plants:StaticMenu.PlantsEntry
+  | prerolls:StaticMenu.PrerollsEntry]
+
+[StaticMenu.ApothecaryEntry
+  | key:string
+  | value:opencannabis.products.Apothecary]
+
+[StaticMenu.CartridgesEntry
+  | key:string
+  | value:opencannabis.products.Cartridge]
+
+[StaticMenu.EdiblesEntry
+  | key:string
+  | value:opencannabis.products.Edible]
+
+{% endnomnoml %}
+
+{% nomnoml %}
+
+#fill: #d5e7ee; #8ebff2
+[StaticMenu.ExtractsEntry
+  | key:string
+  | value:opencannabis.products.Extract]
+
+[StaticMenu.FlowersEntry
+  | key:string
+  | value:opencannabis.products.Flower]
+
+
+[StaticMenu.MerchandiseEntry
+  | key:string
+  | value:opencannabis.products.Merchandise]
+
+[StaticMenu.PrerollsEntry
```

Comment:
This should be **StaticMenu.PlantsEntry**

Reply:
Upon further inspection it looked like i had switched prerolls, and merchandise. switched them back around.

Reply:
no i had totally messed it up
good catch